package-lock.json

```diff
@@ -0,0 +1,2176 @@
+{
+  "name": "nemo-screenshot",
+  "version": "2.2.4",
+  "lockfileVersion": 1,
```

Comment:
Do we need to check in package-lock.json?

Reply:
My understanding is that it should be committed, yes.
https://docs.npmjs.com/files/package-lock.json